bin/main.js

```diff
@@ -0,0 +1,51 @@
+const { askToReview } = require("../src/lib/review");
+const { addLabels } = require("../src/lib/label");
+const { isCIRequired } = require("../src/lib/ci");
+const { isFirstPR } = require("../src/lib/pr");
+
```

Comment:
We don't require the files present on the repo to work wth the bot, they are just for examples.

Reply:
Ah okay, nevermind